src/websocketadapter/WebSocketAdapter.ts

```diff
@@ -41,6 +41,14 @@ export default interface WebSocketAdapter {
    */
   addEventListener(handler: string, eventListener: EventListener): void;
 
+  /**
+   * Remove an event listener for one of the handler types.
+   *
+   * @param handler The handler being assigned.
+   * @param eventListener The event listener to add.
+   */
+  removeEventListener(handler: string, eventListener: EventListener): void;
```

Comment:
unnecessary probably but shouldn't this be optional ?

Reply:
A good catch!

Reply:
Fixed.